reddit_io.py

```diff
@@ -28,9 +28,9 @@ class RedditIO(threading.Thread, LogicMixin):
 
 	_praw = None
 
-	_subreddit = 'talkwithgpt2bots'
+	#_subreddit = 'talkwithgpt2bots'
```

Comment:
We can remove all these lines 31-44 including comments because they're redundant.

Reply:
I just left them in for reference when making the ini part. Will do.

Reply:
Actually, I'm gonna save this until the end (once again, for reference while coding.) Resolving for now